code/modules/projectiles/projectile/beams.dm

```diff
@@ -20,6 +20,9 @@
 	muzzle_type = /obj/effect/projectile/muzzle/laser
 	impact_type = /obj/effect/projectile/impact/laser
 
+/obj/item/projectile/beam/laser/smartpistol
+	damage = 30
```

Comment:
Buffed lasers for the smart pistol only?

Reply:
Yep.

Reply:
Ike has yet to learn that 30 burn damage per hit is a lot apparently.